.env.staging

```diff
@@ -0,0 +1,2 @@
+VUE_APP_MODE = staging
```

Comment:
Probably we need a file .env.production with a setting VUE_APP_MODE = production.  I think the vue-cli build process uses  VUE_APP_MODE production.  I don't really know what the mode "staging" is for.

Reply:
We probably do, we just didn't think about production yet (we don't know the URLs). So we used staging. We usually have three environments - development (which is always local), staging (which is a test environment, what you call development I guess) and production.

Reply:
Ok.  Maybe we can put a build stage in package.json for staging then?